router/api.go

```diff
@@ -230,3 +261,27 @@ func (api *AWSServerlessApi) getSwaggerFromString(input string) ([]byte, error)
 func (api *AWSServerlessApi) getSwaggerFromMap(input map[string]interface{}) ([]byte, error) {
 	return json.Marshal(input)
 }
+
+// Recursively convert a map[interface{}]interface{} (yaml) to map[string]interface{} (json)
+// with an additional special case for the Swagger version that makes the offical Swagger
+// library very upset.
+func yamlToJSON(i interface{}) interface{} {
+	switch x := i.(type) {
+	case map[interface{}]interface{}:
+		m2 := map[string]interface{}{}
+		for k, v := range x {
+			// we have a special case for the swagger version, we need to convert it to a string
```

Comment:
Out of curiosity, where is this workaround requirement coming from? 

Is it because the official swagger library isn't spec compliant?
Is it because it's stricter than a lot of other libraries, so templates people have that work elsewhere don't work here?

Reply:
In the Swagger specs [this field is a string](https://github.com/go-openapi/spec/blob/master/swagger.go#L75). However, unless you quote it, the yaml parser reads it as a numeric and it will need explicit casting when I load it into the Swagger object. Most people forget to quote it in yaml.

Reply:
👍

Reply:
I should have said here, optimizing for devex (users of the CLI) over cleanliness of our code since this is a single-purpose function in here.